### PR TITLE
Convert the project to the RISC-V docs-template (Try #2)

### PR DESCRIPTION
**Pull request by jjscheel**

### CODE REVIEW

**Comment by jjscheel** [comment]
We now appear to be failing because the autobuild cannot find the themes
```
rm -f riscv-configuration-structure-draft.pdf riscv-configuration-structure-release.pdf
asciidoctor-pdf \
-a toc \
-a compress \
-a pdf-style=resources/riscv-themes/riscv-pdf.yml \
-a pdf-fontsdir=resources/riscv-fonts \
-o riscv-configuration-structure-draft.pdf riscv-configuration-structure-draft.adoc
No such file or directory @ rb_sysopen - resources/riscv-themes/riscv-pdf.yml
  Use --trace for backtrace
make: *** [Makefile:13: riscv-configuration-structure-draft.pdf] Error 1
Error: Process completed with exit code 2.
```
@cetola, is this perhaps due to priming of the sub-modules in the build environment?

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,13 @@ draft:	$(DRAFT).pdf
...
 
 %.pdf: %.adoc
-	asciidoctor-pdf $<
+	asciidoctor-pdf \
+	-a toc \
+	-a compress \
+	-a pdf-style=resources/riscv-themes/riscv-pdf.yml \
+	-a pdf-fontsdir=resources/riscv-fonts \
+	-o $@ $<
+#	asciidoctor-pdf $<
```
Comment (timsifive) [inline Makefile:19]: Can you remove this line?

**Review by timsifive — COMMENTED** [review]
Presumably the action needs to be updated to checkout submodules. https://remarkablemark.org/blog/2021/02/21/github-actions-checkout-submodules/ gives examples on how that works.

**Comment by jjscheel** [comment]
Not happy with PR.  Closing.  Will re-submit.

**Pull request closed without merging**